packages/nocodb/tests/unit/rest/tests/meta-apis/base.test.ts

```
@@ -0,0 +1,748 @@
+        const baseUser = baseUsers.find(
+          // @ts-expect-error type not present, but data present
+          (bu) => bu.id === responseBaseUser.id,
+        )!;
+        if (responseBaseUser.roles === 'owner') return; // skip testing for org level user;
+        const underlyingUser = users.find((u) => u.id === responseBaseUser.id)!;
+
+        matchBaseUser(responseBaseUser, baseUser, underlyingUser);
+      });
+    });
+
+    it.only('Create, Update and Delete Base Users', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)